CodeEdit/Features/Documents/WorkspaceDocument.swift

```diff
@@ -111,6 +111,8 @@ final class WorkspaceDocument: NSDocument, ObservableObject, NSToolbarDelegate {
 
     private func initWorkspaceState(_ url: URL) throws {
         self.fileURL = url
+        self.displayName = url.lastPathComponent
```

Comment:
Jus to be clear, this will show nhe filename from tho whole path/URL to the file right?

Reply:
Yes it will, just the last part of it.